src/tree/v2/azure/SubscriptionItem.ts

```diff
@@ -25,10 +26,19 @@ export class SubscriptionItem implements ResourceGroupsItem {
         private readonly context: ResourceGroupsTreeContext,
         private readonly resourceGroupingManager: AzureResourceGroupingManager,
         private readonly resourceProviderManager: AzureResourceProviderManager,
-        public readonly subscription: AzureSubscription) {
+        subscription: AzureSubscription) {
+
+        this.subscription = {
+            // for v1.5 compatibility
+            ...createSubscriptionContext(subscription),
```

Comment:
Are there any property collisions?

Reply:
The types have overlapping properties, but they all have the same types. So it should be fine.